climpred/tests/test_loadutils.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import xarray as xr
+
+
+def test_open_dataset_locally():
+    """Opens all files listed in file_alias_dict."""
+    from climpred.loadutils import file_alias_dict as datasets
```

Comment:
I would bump this up to the header to follow standard practices. I.e., don't import within a function.